docs/appendices/release-notes/5.8.4.rst

```diff
@@ -47,6 +47,14 @@ See the :ref:`version_5.8.0` release notes for a full list of changes in the
 Fixes
 =====
 
+- Fixed an issue that caused wrong results when a table function was combined
+  with a UNION ALL e.g.::
+      SELECT a, b FROM t1
```

Comment:
you'd need an empty line between `::` and the query.

Reply:
👍